site/content@beta/backend-apis/core-info.md

```diff
@@ -0,0 +1,155 @@
+
+<!-- ### Automatic routing
+
+The diagram below present the flow of automatic routing.
+
+![Automatic routing](automatic-routing.png "automatic routing")
+
+### Manual routing
+
+The diagram below present the flow of manual routing.
+
+![Manual routing](manual-routing.png "manual routing") -->
+
+<!-- 1. Czy dajemy te grafiki, przerabiamy je? 
+
+    2. Czy wspominamy o custom routingu, ktory bedzie mozliwy w przyszlosci? 
+    
+    3. Czy rezygnujemy z opisow i linkujemy do artykulu w Help Center? 
+    
+    4. Fajnie by bylo tu kiedys nagrac filmik z lektorem  -->
+
+### Router system messages
+
+While the chat changes its state, the router sends system messages. Read more about [system messages](#system-message).
+
+<!-- W tym zdaniu chodzi o to, ze routing zmienia stan chatow? Przerobilam tak jak mi sie wydaje -->
+
+####  Overview
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat is unassigned because <agent_name> hasn't replied in <minutes_number> minutes` | `routing.unassigned` |
+| `Chat assigned to <agent_name> because <agent_name> hasn't replied in <minutes_number> minutes` | `routing.assigned` |
+| `Chat archived due to long inactivity` | `routing.archived` |
+| `Chat is idle due to <minutes_number> minutes of inactivity` | `routing.idle` |
+| `Chat archived because customer was banned by <agent> for N days` | `customer_banned` |
+
+#### System messages: rounting.assigned
+
+##### General info
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat assigned to <agent_name>` | `routing.assigned` |
+
+##### Use cases
+
+| Case| Routing type |
+|--------------|---------------------|
+| A new chat is started and an agent is available | automatic |
+| An agent has left the chat and another agent is available | automatic |
+| An agent got available for an unassigned chat | automatic |
+
+#### System messages: rounting.unassigned
+
+##### General info
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat is unassigned` | `routing.unassigned` |
+
+##### Use cases
+
+| Case| Routing type |
+|--------------|---------------------|
+| An agent has left the chat and there were no other agents are available | automatic |
+| No free agent slots available | automatic |
+| A chat is unassigned | manual |
+
+## Bot Agents
+
+Bot Agents are similar to their human counterparts. They can join chats and post messages, but they also have a special feature: you can attach [webhooks](../configuration-api/#webhooks) to them.
+
+<img src="images/bot-agent.jpg" alt="LiveChat Bot Agent" class="has-border"/>
+
+Bot Agents are created and managed with the use of the  [Configuration API](../configuration-api/#bot-agent). They communicate with the [Agent Chat API](../agent-chat-api/) by the [**Web API**](../agent-chat-api/#web-api) or **websocket connection**. They listen to incoming webhooks (or pushes) and react to them.
```

Comment:
Do "Websocket connection" podrzuciłbym ten link: ../agent-chat-api/#real-time-messaging-api

Reply:
Zastanawiam się czy jak bedziemy przerabiac  Agent Chat API i Customer Chat API to nie usuniemy z nich teoretycznych opisów, np. o RTM API. Wtedy W Agent/Customer Chat API wspomnialoby sie o RTM i dalo link do Backend APIs, gdzie to jest szerzej opisane. Tak, żeby Backend APIs skupiał się na teorii, a Agent/Customer Chat API na praktycznych przykładach.

Reply:
Dam tu link do sekcji, która jest na samym początku czyli **RTM API vs. Web API**